src/components/popup/PopupDirective.js

```diff
@@ -71,18 +76,15 @@ goog.require('ga_window_service');
           scope.options.isReduced = false;
 
           // Per default hide the print function
-          if (!angular.isDefined(scope.options.showPrint) ||
-              gaBrowserSniffer.mobile) {
+          if (!angular.isDefined(scope.options.showPrint)) {
```

Comment:
This will now show the print button on mobile devices - which is not the case now and doesn't really make sense for mobile devices.

We could say that this is acceptable. Thoughts?